goblet/resources/routes.py

```diff
@@ -381,11 +381,22 @@ def add_route(self, entry):
 
         if entry.form_data:
             params.append({"in": "formData", "name": "file", "type": "file"})
+
+        if entry.query:
+            if isinstance(entry.query, dict):
+                for query in entry.query.params:
+                    params.append(
+                        {
+                            "in": "query",
```

Comment:
I think we can make it more of a pass through since there are other possible params as well such an min, default, etc. So i think just make sure kwarg, query is a list of params, where each param is a dict type. 

Otherwise this is good. Just make sure to add a test to make sure the spec is created as expected. See similar test case for request body

Reply:
Also would be nice to add an example in docs + example folder